src/lib/market-service/portals/portals.ts

```diff
@@ -111,8 +111,7 @@ export class PortalsMarketService implements MarketService {
 
           const params = {
             limit: Math.min(this.defaultGetByMarketCapArgs.count, argsToUse.count),
-            minLiquidity: '1000',
-            minApy: '1',
+            minLiquidity: '100000',
```

Comment:
Can't do the `minApy` too thing since LP-YCRV this was spotted for has 0 APY, but still quite the liquidity